nmdc_runtime/api/endpoints/metadata.py

```diff
@@ -113,9 +123,26 @@ async def submit_changesheet(
 
 @router.get("/metadata/stored_files/{object_id}")
 async def get_stored_metadata_object(
-    object_id: str,
+    object_id: Annotated[
+        str,
+        Path(
+            title="Metadata file ObjectId",
+            description="The ObjectId (`_id`) of the metadata file you want to get.\n\n_Example_: `507f1f77bcf86cd799439011`",
+            examples=["507f1f77bcf86cd799439011"],
+        ),
+    ],
     mdb: MongoDatabase = Depends(get_mongo_db),
 ):
+    r"""
+    Undocumented
```

Comment:
Hi @dwinston, will you summarize what this API endpoint was designed to do? I don't have any experience with GridFS and don't know how our users would get an ObjectId for a GridFS file (which I think is what the `object_id` parameter was designed to contain).

Reply:
done via e00c4bb11b1869041e08e4dd5f751680bf79fb1a